app/services/data/auto-approve-claim-expenses.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../../../knexfile').asyncworker
+const knex = require('knex')(config)
+
+const statusEnum = require('../../constants/status-enum')
+
+module.exports = function (claimExpenses) {
+  var updates = []
+  claimExpenses.forEach(function (claimExpense) {
+    var updatedClaimExpense = {
+      Status: statusEnum.AUTOAPPROVED,
```

Comment:
use APPROVED not AutoApproved, as AutoApproved may mess with payment/notification logic